packages/cli/src/commands/eject/index.ts

```diff
@@ -18,7 +18,7 @@ interface PackageJson {
 }
 
 const TUTORIALKIT_VERSION = pkg.version;
-const REQUIRED_DEPENDENCIES = ['@tutorialkit/runtime', '@webcontainer/api', 'nanostores', '@nanostores/react'];
+const REQUIRED_DEPENDENCIES = ['@tutorialkit/runtime', '@webcontainer/api', 'nanostores', '@nanostores/react', 'kleur'];
```

Comment:
After moving the CLI test from `npm` to `pnpm` it revealed a bug in our `eject` command. :tada: